cristin-organization/src/test/java/no/unit/nva/cristin/organization/organization.feature

```diff
@@ -59,3 +60,19 @@ Feature: API tests for Cristin Organization retrieve and search
     And match response.hits == '#array'
     And match response.size == '#number'
     And match response.hits == '#[2]' // hits array length == 0
+
```

Comment:
Should you add a test with depth = none?

Reply:
yes, done - as i should be in the first place